src/Core/RevEng.Core.60/Routines/Functions/SqlServerFunctionModelFactory.cs

```diff
@@ -44,11 +44,14 @@ protected override List<List<ModuleResultElement>> GetResultElementLists(SqlConn
             var sql = $@"
 SELECT 
     c.name,
-    COALESCE(type_name(c.system_type_id), type_name(c.user_type_id)) AS type_name,
+    t.name AS type_name,
```

Comment:
This seems wrong

Reply:
why ? its the type name i verified this change

Reply:
But it can be either a user_type or a system type, but I only see this join:

`inner JOIN sys.types AS t ON c.user_type_id = t.user_type_id`

maybe you can add another:

`inner JOIN sys.types AS st ON c.system_type_id = t.system_type_id`

and then leave the COALESCE

Reply:
so maybe it should be two left join because the COALESCE tell us ita can be null